pkg/plugins/optional/helm/v1alpha/scaffolds/init.go

```diff
@@ -71,16 +71,17 @@ func (s *initScaffolder) Scaffold() error {
 
 	imagesEnvVars := s.getDeployImagesEnvVars()
 
+	scaffold := machinery.NewScaffold(s.fs,
+		machinery.WithConfig(s.config),
+	)
+
+	// Found webhooks by looking at the config our scaffolds files
 	mutatingWebhooks, validatingWebhooks, err := s.extractWebhooksFromGeneratedFiles()
 	if err != nil {
 		return fmt.Errorf("failed to extract webhooks: %w", err)
 	}
+	hasWebhooks := hasWebhooks(s.config) || (len(mutatingWebhooks) > 0 && len(validatingWebhooks) > 0)
```

Comment:
Wondering if it’d be okay to change the name of the boolean variable so it’s not the same as the function `hasWebhooks`
Or, alternatively, update the function name instead, maybe something like `webhooksConfigured`.
(Just in case this would help readability.)

Reply:
Thank you. 
I changed the func name for hasWebhooksWith which seems the most appropriate.